app/src/main/java/chat/rocket/android/activity/ServerConfigActivity.java

```diff
@@ -77,14 +83,27 @@ protected void onPause() {
   }
 
   private void onRenderServerConfigError(ServerConfig config) {
-    if (config != null) {
+    if (config.getState() == ServerConfig.STATE_CONNECTION_ERROR) {
```

Comment:
And without the change to the query in the observer (around line 57) this block is called a lot of times.... like in a loop forever. So I changed it in order to stop that behavior. But it will be called a couple of times.